core/src/main/java/io/aiven/klaw/service/UiConfigControllerService.java

```diff
@@ -66,27 +68,29 @@ public List<ActivityLog> showActivityLog(String env, String pageNo, String curre
         currentPage,
         origActivityList,
         (pageContext, activityLog) -> {
-          activityLog.setEnvName(
+          ActivityLogModel activityLogModel = new ActivityLogModel();
+          copyProperties(activityLog, activityLogModel);
+          activityLogModel.setEnvName(
               getEnvName(activityLog.getEnv(), activityLog.getActivityName(), tenantId));
-          activityLog.setDetails(activityLog.getDetails().replaceAll("null", ""));
-          activityLog.setAllPageNos(pageContext.getAllPageNos());
-          activityLog.setTotalNoPages(pageContext.getTotalPages());
-          activityLog.setCurrentPage(pageContext.getPageNo());
-          activityLog.setTeam(
+          activityLogModel.setDetails(activityLog.getDetails().replaceAll("null", ""));
+          activityLogModel.setAllPageNos(pageContext.getAllPageNos());
+          activityLogModel.setTotalNoPages(pageContext.getTotalPages());
+          activityLogModel.setCurrentPage(pageContext.getPageNo());
+          activityLogModel.setTeam(
               manageDatabase.getTeamNameFromTeamId(tenantId, activityLog.getTeamId()));
-          return activityLog;
+          return activityLogModel;
         });
   }
 
   public String getEnvName(String envId, String activityName, int tenantId) {
     Optional<Env> envFound;
 
-    if ("SchemaRequest".equals(activityName)) {
+    if ("SchemaRequest".equalsIgnoreCase(activityName)) {
```

Comment:
I know this is legacy but we probably should have these "SchemaRequest" and "ConnectorRequest" as class constants if we dont have an enum for them

Reply:
Nice suggestion. I flipped the condition abit